lib/Less/Functions.php

```diff
@@ -813,11 +813,11 @@ private function _isa($n, $type) {
 		return is_a($n, $type) ? new Less_Tree_Keyword('true') : new Less_Tree_Keyword('false');
 	}
 
-	public function tint($color, $amount) {
+	public function tint($color, $amount = null) {
```

Comment:
Even though the default value of `$amount` of `Less_Functions::mix()` is also 50, you shouldn't leave `Less_Functions::tint()`'s default `$amount` up to `Less_Functions::mix()`. Just think that less.js changes the default `$amout` of `Less_Functions::mix()` to e.g. 66.67 - it would cause an unexpected side-effect.